packages/babel-code-frame/test/index.js

```diff
@@ -105,8 +105,8 @@ describe("@babel/code-frame", function () {
   test("opts.highlightCode with multiple columns and lines", function () {
     // prettier-ignore
     const rawLines = [
-      "function a(b, c) {", 
```

Comment:
oops

Reply:
ergh, shouldn't matter, I thought for a moment the trailing spaces were inside the string. So I'll leave it as is for now unless you want me to roll back

Reply:
I'm surprised that it wasn't already fixed by prettier :thinking:

Reply:
the ignore above it I assume 🙂